src/toast/schedule.py

```diff
@@ -371,27 +371,31 @@ def _parse_line(line):
                 isplit, nsplit = file_split
             scan_counters = dict()
 
+            read_header = False
```

Comment:
These changes alter the logic. Earlier, comment lines were optional but now the header only gets read _if_ there is at least one comment line before the header.

Reply:
Good point, will fix.  At some point I would like to switch formats to HDF5 like the satellite schedule, but that would break use of existing schedule files.  We can figure out the least disruptive time for that.